css/tablet.css

```diff
@@ -0,0 +1,30 @@
+.hero-header {
+    display: grid;
+    place-items: center;
+    background-image: url('/img/hero-md.jpg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
```

Comment:
Please use the same box model values as the star channel website does
![Screen Shot 2022-02-27 at 6 38 50 PM](https://user-images.githubusercontent.com/36536646/155904862-ecd341a3-52a8-462b-b397-f3e33676e197.png)

Reply:
Look what happen in the tablet viewport.

![Screen Shot 2022-02-27 at 6 39 49 PM](https://user-images.githubusercontent.com/36536646/155904892-8a10a21f-27eb-48ed-988f-6f0c48df409a.png)